### ISSUE TITLE
AVT 3 - Carbon X Notification Component example is not a fully working example

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1825]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)
<and>
Windows 7 with JAWS 
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
AVT 3 - The notification examples are not fully functional so they do not make sense to a screen reader. The notifications display as static text, so we are unable to confirm the message is announced correctly.
The expectation is inline notifications display inline as intended and the Toast notification should display the message and disappear within 5 seconds so they can be fully tested. 

The examples use color and an icon to indicate status, there should be some other way such as text so screen reader users  can understand the status as well.  [See WCAG SC 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html).

**Comment by dakahn** [comment]
Providing these examples as working _in app_ with complete functionality is not in the scope of this Storybook. That said there is a need for implementation details and documentation from a a11y standpoint specifically and that's forthcoming.
